apps/web/src/components/split-pane/index.tsx

```diff
@@ -96,14 +96,15 @@ export const SplitPane = React.forwardRef<
   const wrapSize = useRef(0);
   const childrenLength = childrenToArray(children).length;
   const autoSaveKey = autoSaveId ? `csp:${autoSaveId}` : undefined;
+  const previousSize = useRef(0);
```

Comment:
The naming could be improved.

```suggestion
  const lastCollapsedPaneSize = useRef(0);
```